Jenkinsfile

```diff
@@ -33,6 +46,7 @@ pipeline {
         }
         stage('Publish') {
             when {
+                expression { env.JAVA_VERSION == 'java8' }
```

Comment:
I put this in the matrix so it can keep going on the java8 node it used to build.

(Not publishing a java11 build because that sounds like a different kettle of worms.)

But maybe this should be outside the matrix so it doesn't happen unless the java11 tests succeed?